tlslite/tlsconnection.py

```diff
@@ -773,6 +776,9 @@ def _clientGetServerHello(self, settings, clientHello):
                         AlertDescription.illegal_parameter,
                         "Server selected ALPN protocol we did not advertise"):
                     yield result
+        if settings.useHeartbeatExtension and \
+                serverHello.getExtension(ExtensionType.heartbeat):
```

Comment:
How can I get, if the mode, what I got from opposite site, is in `HeartbeatMode` class? I mean is any best practice, how I can iterate through class atributes and compare them with value?